README.md

```diff
@@ -1,2 +1,24 @@
 # Hello Microverse test
- test for fix com
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
```

Comment:
- [ **OPTIONAL** ] Kindly remove this unnecessary text as your project has already the title. I can suggest you move the **Project Name** from the first line and replace it at his line. This will make your documentation more organized.